tsz-compress/src/v2/encode.rs

```diff
@@ -76,10 +67,10 @@ fn push_three_bits(q: &mut CompressionQueue<10>, buf: &mut HalfVec) {
     let mut word: u32 = 0;
     let values = q.pop_n::<N>();
     for i in 0..N1 {
-        word |= values[i];
+        word |= values[i] as u32;
```

Comment:
It should be more efficient to build these up in the word as usize and cast once when creating the Full